spec/javascripts/support/jasmine.yml

```diff
@@ -13,6 +13,7 @@
 src_files:
   - assets/application.js
   - assets/winners-bundle.js
+  - assets/jquery.js
```

Comment:
jquery should probably go first on this list of assets, just in case it matters for jasmine's loading